doc/source/whatsnew/v0.25.1.rst

```diff
@@ -72,6 +72,7 @@ I/O
 
 - Avoid calling ``S3File.s3`` when reading parquet, as this was removed in s3fs version 0.3.0 (:issue:`27756`)
 - Better error message when a negative header is passed in :func:`pandas.read_csv` (:issue:`27779`)
+- :func:`read_sas()` now handles dates and datetimes larger than pd.Timestamp.max returning them as datetime.datetime objects (:issue:`20927`)
```

Comment:
pd.Timestamp.max should be something like :attr:`\Timestamp.max\`.

datetime.datetime should be :class:\`date time.datetime`.

Reply:
Hi Tom,

Thanks for the comment, I'll fix the documentation tags.

mroeschke mentioned above that out of bounds datetimes are often handled as Periods
[https://pandas.pydata.org/pandas-docs/stable/user_guide/timeseries.html#representing-out-of-bounds-spans](url)
However there is an issue with Periods [https://github.com/pandas-dev/pandas/issues/28104](url) which prevents me using them at the moment.